src/stories/OSCALBackMatter.stories.js

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+import OSCALBackMatter from "../components/OSCALBackMatter";
+
+export default {
+  title: "Components/Back Matter",
+  component: OSCALBackMatter,
+};
+
+const Template = (args) => <OSCALBackMatter {...args} />;
+
+export const Default = Template.bind({});
+
+export const BackMatterTitle = Template.bind({});
+
+export const BackMatterDescription = Template.bind({});
+
+export const BackMatterCitation = Template.bind({});
+
+export const BackMatterMediaType = Template.bind({});
+
+const exampleBackMatter = {
+  resources: [
+    {
+      uuid: "dc380596-027f-423b-83f2-82757554ee27",
+      description:
+        "NIST Special Publication 800-53 Revision 4: Security and Privacy Controls for Federal Information Systems and Organizations",
+      rlinks: [
+        {
+          href: "NIST_SP-800-53_rev4_catalog.json",
+          "media-type": "application/oscal.catalog+json",
+        },
+      ],
+    },
+  ],
+};
+
+const exampleBackMatterTitle = {
+  resources: [
+    {
+      uuid: "dc380596-027f-423b-83f2-82757554ee27",
+      title: "Resource Title",
+      rlinks: [
+        {
+          href: "NIST_SP-800-53_rev4_catalog.json",
+        },
+      ],
+    },
+  ],
+};
+
+const exampleBackMatterDescription = {
```

Comment:
Rather than repeating most of the data in each example, should the extended examples just add to the default?

We could do something like:
```javascript
const exampleBackMatterDescription = () => {
  const example = exampleBackMatter;
  example.resources[0].description = "some description";
  return example;
};
```
There may be a more elegant way to represent that.

Reply:
Something like this would affect all instances of `exampleBackMatter` because it's passed by reference. I'm not sure of an easy way to do deep copies here, but the back matter test data itself seems to avoid duplication pretty well already.

Reply:
Yes, that's why I mentioned just the "extended" examples, i.e. additive.  We could do deep copies for simple objects with JSON.parse and stringify though that can break down quickly with certain property types, or use a library like lodash, but I'm not sure either of those approaches would be worth it in this case.

I'm also fine with the current duplication, was just a thought.